php/Blockonomics.php

```diff
@@ -702,6 +717,24 @@ public function get_order_info($order_id, $crypto){
         exit(json_encode($order));
     }
 
+    // Get the order info by id and crypto
+    public function get_order_amount_info($order_id, $crypto){
+        $order = $this->process_order($order_id, $crypto);
+
+        $order_amount = $this->fix_displaying_small_values($order['satoshi']);
+        
+        $cryptos = $this->getActiveCurrencies();
+        $crypto_obj = $cryptos[$crypto];
+
+        $response = array(
+            "payment_uri" => $this->get_crypto_payment_uri($crypto_obj, $order['address'], $order_amount),
```

Comment:
Yes, it can be, but having it form at two places when it can be formed at the same place doesn't look good to maintain. We'll have to update it anytime we decide to change something regarding it in both JS and PHP.